readthedocs/projects/forms.py

```diff
@@ -246,7 +246,7 @@ def __init__(self, *args, **kwargs):
         # the special cases of LATEST and STABLE.
         all_versions_choices = [
             (v.commit_name, v.verbose_name)
```

Comment:
We can use `verbose_name` for the id too now (commit_name was used only for the special cases of latest and stable). Also, we should add a test when the user has a version named `latest` or `stable`. Using `commit_name` will make the test fail.

Something like https://github.com/rtfd/readthedocs.org/pull/5523/commits/82944cad1d1d7e6a300aa5cc55666196791d5950

Reply:
Okay I'll Update the PR accordingly. :+1:

Reply:
@stsewd What did you actually mean by this can you please clarify?

> Using commit_name will make the test fail.

Reply:
If you write a test with a tag named `latest` or `stable` the current implementation will fail.

Reply:
I mean, a version created by the user `machine=False`

Reply:
@stsewd Updated the changes. but I'm Not sure I correctly understood what you meant about the tests. please have a look and let me know.